gremlin/traversal/flow_test.go

```diff
@@ -132,8 +122,8 @@ func TestHasStep(t *testing.T) {
 		time.Sleep(100 * time.Millisecond)
 	}
 
-	flowChan <- &flow.Operation{Type: flow.ReplaceOperation, Flow: newICMPFlow(222), Key: strconv.Itoa(rand.Int())}
-	flowChan <- &flow.Operation{Type: flow.ReplaceOperation, Flow: newICMPFlow(444), Key: strconv.Itoa(rand.Int())}
+	flowChan <- newICMPFlow(222, "node1", "", "")
```

Comment:
please do not change the test, please add epbf specific tests

Reply:
done